### PR TITLE
Deprecate REX time span option

### PR DESCRIPTION
**Pull request by adamruzicka**
A companion to https://github.com/theforeman/foreman_remote_execution/pull/834

Opening as a draft until the REX PR goes in.

### CODE REVIEW

**Comment by adamruzicka** [comment]
Should I also regenerate all the apidoc data and fixtures?

**Comment by evgeni** [comment]
NACK-ish.
We try to support older versions too.
Can you document that the option is removed in REX version something and we'll remove it at some point in the future?
(When the apidoc doesn't expose a param that the user has set, the param won't be sent and a warning is displayed that the param is not supported)

**Comment by adamruzicka** [comment]
Ah, I didn't know that, thank you. Will do

File: plugins/modules/job_invocation.py [hunk 1]
```diff
@@ -117,6 +117,7 @@
       time_span:
         description:
           - Distribute tasks over given number of seconds
+          - This is deprecated with foreman_remote_execution-11.0.0
```
Comment (evgeni) [inline plugins/modules/job_invocation.py]: deprecated or removed? the PR you linked fully removes that parameter from the API

**Review by evgeni — APPROVED** [review]

**Comment by adamruzicka** [comment]
We got the green light, time span is going to go away

**Pull request merged**